fandogh_cli/service_commands.py

```diff
@@ -18,16 +18,18 @@ def service():
 @click.option('--env', '-e', 'envs', help='Environment variables (format: VARIABLE_NAME=VARIABLE_VALUE)', multiple=True)
 @click.option('--hosts', '-h', 'hosts', help='Custom hosts that service should be accessible through', multiple=True)
 @click.option('--port', '-p', 'port', help='The service port that will be exposed on port 80 to worldwide', default=80)
+@click.option('--registry_secret', '-r', 'registry_secret',
```

Comment:
-r is more suitable for replica. 
usually in long options dash `-` is used to concat the parts instead of underscore.